pretix_zugferd/invoice.py

```diff
@@ -370,6 +371,19 @@ def _zugferd_generate_document(self, invoice):
         doc.trade.settlement.monetary_summation.tax_total = (taxtotal, cc)
         doc.trade.settlement.monetary_summation.grand_total = total
         doc.trade.settlement.monetary_summation.due_amount = total
+
+        if not self.invoice.is_cancellation:
+            payment_refund_sum = self.invoice.order.payment_refund_sum
+            if payment_refund_sum and (
+                self.invoice.event.settings.invoice_show_payments
+                or self.invoice.order.status == Order.STATUS_PAID
+            ):
+                doc.trade.settlement.monetary_summation.prepaid_total = (
+                    payment_refund_sum
+                )
+                doc.trade.settlement.monetary_summation.due_amount = (
+                    total - payment_refund_sum
```

Comment:
If someone overpaid the order, this creates a negative amount. Technically correct, but I just want to make sure, that that is allowed.